girder/cumulus/plugin_tests/cluster_test.py

```diff
@@ -579,11 +610,15 @@ def test_terminate(self, terminate_cluster):
         status_body = {
             'status': 'running'
         }
+        r = self.request('/clusters/%s' %
+                         str(cluster_id),
+                         type='application/json', method='PATCH',
+                         body=json.dumps(status_body), user=self._user)
 
```

Comment:
We can't terminate a 'created' cluster so we patch the status of the cluster to 'running' so the validation works.